tests/common/mac/ProjectTestHelpers.cs

```diff
@@ -573,7 +574,14 @@ static string GenerateOutputCommand (string tmpDir, Guid guid)
 
 		public static void NugetRestore (string project)
 		{
-			var rv = ExecutionHelper.Execute ("nuget", $"restore {StringUtils.Quote (project)}", out var output);
+			string rootDirectory = FindRootDirectory ();
+
+			Environment.SetEnvironmentVariable ("TargetFrameworkFallbackSearchPaths", rootDirectory + "/Library/Frameworks/Mono.framework/External/xbuild-frameworks");
+			Environment.SetEnvironmentVariable ("MSBuildExtensionsPathFallbackPathsOverride", rootDirectory + "/Library/Frameworks/Mono.framework/External/xbuild");
+			Environment.SetEnvironmentVariable ("XAMMAC_FRAMEWORK_PATH", rootDirectory + "/Library/Frameworks/Xamarin.Mac.framework/Versions/Current");
+			Environment.SetEnvironmentVariable ("XamarinMacFrameworkRoot", rootDirectory + "/Library/Frameworks/Xamarin.Mac.framework/Versions/Current");
```

Comment:
Path.Combine over the concatenation?

Reply:
This is very specifically for macOS, so I don't see a value in using `Path.Combine` here, unless I break the fixed path into it's parts. Also, this is copy pasted from existing code (not necessarily a good reason for sticking to this though). If more people feel this is required, then I can make the change.

Reply:
Nor a big deal. Just doing a review from a general point of view :)